app/client/src/components/pages/WaterbodyReport.js

```diff
@@ -1256,32 +1280,42 @@ function WaterbodyReport({ fullscreen }: Props) {
                       )}
                       {documents.status === 'success' && (
                         <>
-                          {documents.length > 0 && (
+                          {documents.data.length > 0 && (
                             <div css={[boxSectionStyles, { paddingBottom: 0 }]}>
                               <p css={introTextStyles}>
                                 <em>Links below open in a new browser tab.</em>
                               </p>
                             </div>
                           )}
                           <div css={boxSectionStyles}>
-                            <ul css={listStyles}>
+                            <ul css={listStyles} className="document-list">
```

Comment:
```suggestion
                            <ul css={listStyles}>
```
Similar to the comment above, the `document-list` class is not needed here.